### PR TITLE
adds groups and roles list pages (sadly seed data has neither so they…

### PR DESCRIPTION
**Pull request by AllenBW**
… don't show up quite yet)  also new build for those who cant OH also includes tagging match counts

### CODE REVIEW

File: src/client/app/resources/groups.factory.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+(function() {
+  'use strict';
+
+  angular.module('app.resources')
+    .factory('Groups', GroupsFactory);
```
Comment (stackus) [inline src/client/app/resources/groups.factory.js]: Going with singular resource names to align with Rails singular model names.

File: src/client/app/resources/roles.factory.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+(function() {
+  'use strict';
+
+  angular.module('app.resources')
+    .factory('Roles', RolesFactory);
```
Comment (stackus) [inline src/client/app/resources/roles.factory.js]: singular

File: src/client/app/states/admin/groups/list/_list.sass [hunk 3]
```diff
@@ -10,6 +9,167 @@
...
+      width: 116px
+
+    +modifier(padding)
+      width: 20px
+
+    +modifier(description)
+      padding-left: 32px
+      padding-right: 20px
+
+    +modifier(setup)
```
Comment (stackus) [inline src/client/app/states/admin/groups/list/_list.sass]: Do alerts have columns with modifiers like setup, monthly, and hourly?

File: src/client/app/states/admin/roles/list/_list.sass [hunk 4]
```diff
@@ -10,6 +9,167 @@
...
+      width: 116px
+
+    +modifier(padding)
+      width: 20px
+
+    +modifier(description)
+      padding-left: 32px
+      padding-right: 20px
+
+    +modifier(setup)
```
Comment (stackus) [inline src/client/app/states/admin/roles/list/_list.sass]: these are needed?

File: src/client/app/states/admin/roles/list/list.html [hunk 5]
```diff
@@ -1,12 +1,57 @@
...
+    </tr>
+    </thead>
+    <tbody>
+    <tr class="roles-table__row" ng-repeat="row in vm.roles">
+      <td class="roles-table__cell roles-table__cell--padding"></td>
+      <td class="roles-table__cell roles-table__cell--order">{{ ::row.id }}</td>
+      <td class="roles-table__cell roles-table__cell--question">{{ ::row.name}}</td>
+      <td class="roles-table__cell roles-table__cell--question">{{ ::row.description }}</td>
+      <td class="roles-table__cell roles-table__cell--question">
+        <span ng-repeat="permission in row.permissions">{{permission}}, </span>
```
Comment (stackus) [inline src/client/app/states/admin/roles/list/list.html]: It'd be better to have a method on Role that will return the list of permissions joined with a comma.

File: src/client/app/states/admin/roles/list/_list.sass (hunk 4 above)
Comment (AllenBW) [inline src/client/app/states/admin/roles/list/_list.sass]: Padding yes, others likely not, will refactor later

Comment (AllenBW) [inline src/client/app/states/admin/roles/list/_list.sass]: Roger that!! 😃😊

**Pull request merged**